public/app/plugins/datasource/elasticsearch/components/QueryEditor/QueryEditorRow.tsx

```diff
@@ -0,0 +1,91 @@
+import { GrafanaTheme } from '@grafana/data';
+import { IconButton, stylesFactory, useTheme } from '@grafana/ui';
+import { getInlineLabelStyles } from '@grafana/ui/src/components/Forms/InlineLabel';
+import { css, cx } from 'emotion';
+import { noop } from 'lodash';
+import React, { FunctionComponent } from 'react';
+
+const flex = css`
+  label: QueryEditorRow;
+  display: flex;
+  align-items: start;
+  flex-direction: row;
+`;
+
+interface Props {
+  label: string;
+  onRemoveClick?: false | (() => void);
+  onHideClick?: false | (() => void);
+  hidden?: boolean;
+}
+
+export const QueryEditorRow: FunctionComponent<Props> = ({
+  children,
+  label,
+  onRemoveClick,
+  onHideClick,
+  hidden = false,
+}) => {
+  const theme = useTheme();
+  const styles = getStyles(theme);
+
+  return (
+    <fieldset
+      className={css`
+        display: block;
+        margin-bottom: ${theme.spacing.xs};
+      `}
+    >
+      <div className={cx(flex)}>
+        <div className={cx(getInlineLabelStyles(theme, 17).label)}>
+          <legend
+            className={css`
+              font-size: ${theme.typography.size.sm};
```

Comment:
we have some inline styles in this function that can be moved to the getStyles function. (I added it now for the IconButton styles)